Sources/SKCore/BuildSystemList.swift

```diff
@@ -36,20 +36,11 @@ extension BuildSystemList: BuildSystem {
 
   public var indexDatabasePath: AbsolutePath? { return providers.first?.indexDatabasePath }
 
-  public func settings(for url: URL, _ language: Language) -> FileBuildSettings? {
```

Comment:
This is changing behaviour with `BuildSystemList` since the notifications are only implemented for the first one in the list.  We need a new way to inject the fallback settings.

Reply:
This is outdated, BuildSystemList has been removed in a previous PR